Applications/shapeworks/Commands.cpp

```diff
@@ -263,6 +287,34 @@ int SmoothMesh::execute(const optparse::Values &options, SharedCommandData &shar
   return sharedData.mesh.smooth(/*maxRMSErr, numIter*/);
 }
 
+///////////////////////////////////////////////////////////////////////////////
+// Coverage
+///////////////////////////////////////////////////////////////////////////////
+void Coverage::buildParser()
+{
+  const std::string prog = "coverage";
+  const std::string desc = "coverage between two meshes";
```

Comment:
This is not a sufficient description. What does "coverage between two meshes" mean? Maybe _compute_ coverage between two meshes?

Reply:
What is the result? Another mesh?

Reply:
I did not create the coverage command

Reply:
I will ask Praful & Steve how this should be described.